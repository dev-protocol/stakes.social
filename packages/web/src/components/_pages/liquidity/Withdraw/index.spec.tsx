import React from 'react'
import { render } from '@testing-library/react'
import { Withdraw } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/_pages/liquidity/geyser/hooks.ts')
jest.mock('src/fixtures/_pages/liquidity/uniswap-pool/hooks.ts')

describe(`${Withdraw.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Withdraw geyserAddress="0xdummy" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
