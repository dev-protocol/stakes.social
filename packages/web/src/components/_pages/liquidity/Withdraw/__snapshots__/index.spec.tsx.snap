// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Withdraw Snapshot 1`] = `
<body>
  <div>
    <form
      class="ant-form ant-form-vertical sc-jRQBWg kMRzJz"
    >
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Your Reward Multiplier
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            <span
              class="ant-statistic-content-value-int"
            >
              
              1
            </span>
            <span
              class="ant-statistic-content-value-decimal"
            >
              .0
            </span>
          </span>
          <span
            class="ant-statistic-content-suffix"
          >
            X / 5X
          </span>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="sc-bdvvtL lmxjND"
              >
                <label
                  for="amount"
                >
                  Your amount
                </label>
                <span
                  class="ant-input-affix-wrapper ant-input-affix-wrapper-lg sc-gsDKAQ dSVFwg"
                >
                  <input
                    class="ant-input ant-input-lg"
                    id="amount"
                    type="number"
                    value=""
                  />
                  <span
                    class="ant-input-suffix"
                  >
                    <span
                      class="sc-hKwDye hiokQn"
                    >
                      ETHDEV-V2
                    </span>
                    <button
                      class="ant-btn ant-btn-primary ant-btn-sm sc-dkPtRN leCUQt"
                      type="button"
                    >
                      <span>
                        Max
                      </span>
                    </button>
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="sc-bdvvtL lmxjND"
              >
                <label
                  for="estimated"
                >
                  Rewards claimed
                </label>
                <input
                  class="ant-input ant-input-lg ant-input-disabled sc-gsDKAQ dSVFwg"
                  disabled=""
                  id="estimated"
                  type="number"
                  value="0"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item sc-eCImPb kitwYs"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg"
                type="button"
              >
                <span>
                  Withdraw
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
